### PR TITLE
Cms

### PR DESCRIPTION
**Pull request by gshilin**

### CODE REVIEW

File: cms/cms.go [hunk 1]
```diff
@@ -49,21 +49,21 @@ func SyncCMS() {
...
+	Slug    string            `json:"slug"`
+	Title   string            `json:"title"`
+	Content string            `json:"content"`
+	Meta    map[string]string `json:"meta"`
 }
 
-var personsLanguage = regexp.MustCompile("persons-.+?-([a-z]{2})-html")
-var bannersLanguage = regexp.MustCompile(".+-([a-z]{2})")
+var personsLanguage = regexp.MustCompile("^.+?-([a-z]{2})$")
+var bannersLanguage = regexp.MustCompile("^en|he|ru|tr|it|ua|es|de$")
```
Comment (edoshor) [inline cms/cms.go]: please revert to allow future languages

File: api/cms.go [hunk 2]
```diff
@@ -19,10 +19,15 @@ func CMSPerson(c *gin.Context) {
 	if c.Bind(&r) != nil {
 		return
 	}
-	id := c.Param("id")
+	id := c.Query("id")
```
Comment (edoshor) [inline api/cms.go]: why not keep the REST approach ?
path param (`/:id`) instead query params

File: cms/cms.go [hunk 3]
```diff
@@ -132,21 +122,8 @@ func mkdir(permissions os.FileMode, dirs ...string) (err error) {
...
-		return
-	}
-	if err = mkdir(0755, assets, "b", "banners"); err != nil {
-		return
-	}
-	if err = mkdir(0755, assets, "b", "persons"); err != nil {
-		return
-	}
-
+/* Create/clear directories. Return a passive one (i.e. symlink does not point to it) */
```
Comment (edoshor) [inline cms/cms.go]: consider creating a new fresh directory each time instead of a-b.

File: api/cms.go (hunk 2 above)
Comment (gshilin) [reply api/cms.go]: done

File: cms/cms.go (hunk 3 above)
Comment (gshilin) [reply cms/cms.go]: done

File: api/cms.go (hunk 2 above)
Comment (gshilin) [reply api/cms.go]: done

File: cms/cms.go (hunk 3 above)
Comment (gshilin) [reply cms/cms.go]: done

**Pull request merged**